openproblems/tasks/multimodal_data_integration/methods/baseline.py

```diff
@@ -0,0 +1,44 @@
+from ....tools.decorators import method
+from ....tools.normalize import log_cpm
+from ....tools.utils import check_version
+
+import numpy as np
+import sklearn.decomposition
+
+
+@method(
+    method_name="Random Features",
+    paper_name="Random Features (baseline)",
+    paper_url="https://openproblems.bio",
+    paper_year=2022,
+    code_url="https://github.com/openproblems-bio/openproblems",
+    is_baseline=True,
+)
+def random_features(adata, test=False, n_svd=20):
```

Comment:
I would suggest all `random*` method with a `n_times` sample times argument.

Reply:
Not sure I quite understand what you mean. What would we do with that argument?

Reply:
Ah nvm I see your comment above.